containers/Ideas/IdeaProject.js

```diff
@@ -87,11 +87,11 @@ export const IdeaProject = (props) => {
         source={{ uri: bgImage }}
         accessibilityIgnoresInvertColors={true}
       />
+      <LinearGradient
```

Comment:
I'm not super sure about this but it really looks weird if we don't move it here. Or maybe we need to have a negative margin on the scrollview or something